opm/autodiff/BlackoilModelEbos.hpp

```diff
@@ -1423,7 +1423,7 @@ namespace Opm {
                     errlog << ", ...";
                 }
                 errlog << "]";
-                OpmLog::warning("Bubble point numerical problem", errlog.str());
+                OpmLog::info("Bubble point numerical problem", errlog.str());
```

Comment:
I'm not sure I agree with this change. It could point to a problem with the property tables, and therefore be cause to give a warning. Also, what is a good guideline for what should be a warning and what should not be?